src/dmd/cli.d

```diff
@@ -206,7 +211,8 @@ struct Usage
             "generate library rather than object files"
         ),
         Option("m32",
-            "generate 32 bit code"
+            "generate 32 bit code",
+            (TargetOS.all & ~TargetOS.dragonFlyBSD)  // available on all OS'es except DragonFly, which does not support 32-bit binaries
```

Comment:
Not sure if this was an intended use ?
Not sure if it merits an explanation / comment ?

Reply:
Yes that's how it was intended to be used.
BTW `-m32` is also mostly defunct on ArchLinux too as they dropped native support for 32-bit too, but I'm not sure whether it makes sense to complicate the code with such subtleties.

Reply:
Using -m32 with dmd on dragonfly currently does fail (not implemented -> assertion failure). I guess this is not the same on archlinux. They might just lack the 32-bit libraries to compile against.
I can remove it if you want (Or wait for another reviewer).

Reply:
Yes, you can still compile `gcc-multilib` yourself from the AUR (arch user repository).
I don't this being used like you did as I said before, that was intended ;-)

Reply:
OK issue closed